lib/scripts/mtw.js

```diff
@@ -292,6 +301,36 @@ export class ContentScript {
   }
 
   /**
+   * Translate one word in each sentence for a  paragraph.
+   * @param {Object} paragraph - Paragraph nodeType
+   * @param {Object} filteredTMap - filtered translation map
+   * @param {Object} iMap - HTML element for each translated word
+   */
+  translateOneWord(paragraph, filteredTMap, iMap) {
+    for (let i in paragraph.childNodes) {
+      if (paragraph.childNodes[i].nodeType === 3) {
+        if(!/^\s*$/.test(paragraph.childNodes[i].textContent)){
+          let sentences = paragraph.childNodes[i].textContent.split('.');
+          for (let j in sentences) {
+            let words = sentences[j].split(' ');
+            words = _.shuffle(words);
+            for (let k in words) {
```

Comment:
This line can be confusing. Could you add a comment saying that this **for** loop will be interrupted as soon as one word satisfying the conditional below is satisfied. Otherwise, at first glance, it looks like you are translating more than one word per sentence.